README.md

```diff
@@ -1,2 +1,229 @@
+      +materialized: table ##how your marts models will materialized.
+      +dataset: marts ##the subfolder that will be created within your dataset models where the marts will be located.
+      ledger_current_state:
+        +materialized: table  ##how your ledger current state model will materialized.
+        +tags: current_state ##tag used to resource selection syntax.
+```
+
+4. You can also execute the command `dbt debug` to ensure that all configurations are working correctly, and it is possible to initiate data loading and transformation.
+
+## Working with dbt
+
+### Running Models
+
+Executing a dbt model is the same as executing a SQL script. There are many ways to run dbt models: 
+
+1. Run the entire project through `dbt run`
+
+2. Run tagged models, paths or configs through `dbt run --select tag:tag_1 tag_2 tag_3`.
+
+3. Run specific models through `dbt run --select model_1 model_2 model_3`. **Note that this runs only the specified models, unless you follow said model(s) with execution method #4**
+
+4. Run downstream or upstream models by adding a '+' at the beginning(downstream), end(upstream) or both on the model name `dbt run --select +model_1+`
+
+On top of that, dbt supports --excluding, --defer, --target and many other selection types. To see more, please refer to the [documentation](https://docs.getdbt.com/reference/node-selection/syntax).
+
+### Running Tests
+
+Executing tests works the same way as executing runs, with the command `dbt test` accepting many of the same parameters. This will run both schema tests and unique tests, unless commanded otherwise. For more information on the syntax, consult the [documentation](https://docs.getdbt.com/reference/node-selection/syntax).
+
+In dbt, tests come in two ways. Schema tests, which are pre-built macros and can be called in YML schema files, and unique tests, which are user-made and have their own .SQL files, stored in the `tests` folder. Both tests will be executed during a `dbt test`, unless further node selection is provided. A dbt test will fail when the underlying SQL selection returns a result row, being approved when no rows are returned. For further information on tests, please refer to the [documentation](https://docs.getdbt.com/reference/test-configs).
+
+## Project Structure
+
+The Stellar-dbt project follows a staging/marts approach to modelling. The staging step focuses on transforming and preparing the tables for joining on the marts step. In order to diminish cost, all staging .sql files are materialized as ephemeral in the `dbt_project.yml`, allowing code modularity and decoupling while not raising storage costs. 
+
+The marts, on the other hand, are materialized as tables, in order to reduce querying time on the BI and other exposures. It is in this step that tables are joined together to obtain analytical results or rejoin needed information to facilitate date exploration.
+
+### Development Folders
+
+| Name | Description |
+|------|-------------|
```

Comment:
Please remake the table description, these are the internal project descriptions. The public project should have basic descriptions only.